packages/terra-form/tests/jest/components/Input.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Input from '../../../src/components/Input';
+
+it('should render a default component', () => {
+  const input = (<Input value="" />);
+  const wrapper = shallow(input);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a Input with the rest of the props', () => {
+  const input = (
+    <Input
+      required
+      name="foo"
+      value="bar"
+    />);
+
+  const wrapper = shallow(input);
+  expect(wrapper).toMatchSnapshot();
+});
```

Comment:
@emilyrohrbough See how these work for you. From looking at the jest docs, I think this is the best I can get, which I think is ok for our case.

https://github.com/cerner/terra-core/pull/352/commits/f57bf091b3c5985efe503336c3b5fb4c5ae50896